tasks/snapshot.yml

```diff
@@ -58,4 +60,7 @@ tasks:
         git config user.email 'no-reply@lullabot.com'
         git config user.name 'Lullabot Drainpipe'
         git commit -m "Initial commit" --no-gpg-sign
-        git archive --format={{.format}} HEAD
+        git config tar.tar.xz.command "xz -c"
```

Comment:
Interesting we have to set these manually. Is that because the git config is dynamic, and would support us creating a `tar.asdf` file if we defined it?

Reply:
yep!